packages/app-explorer/src/NodeInfo/Summary.tsx

```diff
@@ -45,18 +45,18 @@ class Summary extends React.PureComponent<Props, State> {
 
     return (
       <SummaryBox>
-        <section className='ui--media-medium'>
+        <section>
           <CardSummary label={t('refresh in')}>
             <Elapsed value={nextRefresh} />
           </CardSummary>
-          <CardSummary label={t('total peers')}>
+          <CardSummary label={t('total peers')} className='ui--media-small'>
```

Comment:
(nitpick) With multiple attributes, put each on their own line (in alphabetical order)